signup/package.json

```diff
@@ -13,8 +13,13 @@
     "@chakra-ui/react": "^2.8.1",
     "@emotion/react": "^11.11.1",
     "@emotion/styled": "^11.11.0",
+    "@mailchimp/mailchimp_marketing": "^3.0.80",
```

Comment:
Did you verify this works?

This lib does not work on the frontend, it's a Node package

Reply:
It's just for the `functions` directory (lambda)